cpu/cc2538/startup-gcc.c

```diff
@@ -30,22 +30,23 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 /**
- * \addtogroup cc2538dk
+ * \addtogroup cc2538
  * @{
  *
  * \file
- * Startup code for the cc2538dk platform, to be used when building with gcc
+ * Startup code for the cc2538 chip, to be used when building with gcc
  */
 #include "contiki.h"
 #include "reg.h"
 #include "flash-cca.h"
 #include "sys-ctrl.h"
-#include "uart.h"
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
 extern int main(void);
 /*---------------------------------------------------------------------------*/
+#define WEAK_ALIAS(x) __attribute__ ((weak, alias(#x)))
```

Comment:
How about something like this?

```
#if CC_CONF_WEAK_ALIAS
#define CC_WEAK_ALIAS(f) __attribute__ ((weak, alias(#f)))
#else /* CC_CONF_WEAK_ALIAS */
#define CC_WEAK_ALIAS(f)
#endif /* CC_CONF_WEAK_ALIAS */
```

and then go to the platform's `contiki-conf.h` and add this:

```
#define CC_CONF_WEAK_ALIAS 1
```